### ISSUE TITLE
Merge all SYNK PRS for 3rd Party Apps

### ISSUE THREAD

**Issue by jpvajda** [newrelic/nr1-catalog#241]
We have lot SNYK PRs that are queuing up in our app repos so we should merge those in and close the invalid ones. 

### Acceptance Criteria
- [x] Visit the [App Catalog GH project board](https://github.com/orgs/newrelic/projects/44)
- [x] Review the `In Review` column 
- [x] Close all Snyk PRs that are labeled `invalid` as they are old and have failed a lot of checks.
- [x] merge in all SNYK PRs that pass checks 
- [x] ensure any merge conflicts with package.json or lock files are resolved. 
- [x] If any new SNYK PRs fail, add a `invalid` label and we'll evaluate next steps.

**Comment by rudouglas** [comment]
@jpvajda i followed the steps and made assessments on all new unlabelled PR's. Most of them failed because of the following checks so i'm wondering if they are ok to merge or not? I've labelled these with `question`:

 - `security/snyk`
 - `license/cla`
 - Both

For [nr1-status-page](https://github.com/newrelic/nr1-status-pages/pull/126) there was an ESLinting issue that I was able to resolve, which should resolve the linting fail on 2 other PR's. Who would be best to review this? After that's resolved both `security/snyk` and `license/cla` are also failing so depending on the decision above these can then be resolved too
